components/plugin_marketplace/marketplace_item/marketplace_item.test.tsx

```diff
@@ -36,7 +35,7 @@ describe('components/MarketplaceItem', () => {
             it('when installed version matches available version', () => {
                 const props = {
                     ...baseProps,
-                    installedVersion: baseProps.availableVersion,
+                    installedVersion: baseProps.version,
```

Comment:
I suppose this was a typo. @lieut-data Can you confirm?

Reply:
Indeed, looks like it!